### DISCUSSION TITLE
Displaying the filters all the time

### DISCUSSION THREAD

**Discussion by digitalredefined** [rappasoft/laravel-livewire-tables#772] [Q&A] [answered]
Is there any way to display the filters above the table all the time?  We've only got a single item so it doesn't really make sense for the user to have to click twice to make their selection.

Thanks!

**Comment by rappasoft** [comment, marked as answer]
In v2 you can use a combination of [configurable areas](https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/configurable-areas#content-areas) and [events](https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/events#content-setfilter) to put your own filters in there to control the table.

**Reply by digitalredefined** [reply]
Got it. I'll be upgrading the project in the next iteration, thanks!
